app/javascript/components/cart-products.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div
+        class="debt-product-list"
+      >
+        <div class="debt-product-list__title">
+          <div class="debt-product-list__amount">#</div>
+          <div class="debt-product-list__name">Producto</div>
+          <div class="debt-product-list__total">Precio</div>
+        </div>
+        <div
+          v-for="product in cartProducts"
+          :key="product.id"
+          class="debt-product-list__info"
+          :product="product"
+        >
+          <div class="debt-product-list__amount">{{ product.amount }}</div>
+          <div class="debt-product-list__name">{{ product.name }}</div>
+          <div class="debt-product-list__total">$ {{ product.price }}</div>
+        </div>
+        <div class="debt-total">
+          <div class="debt-total__title">Total (SAT)</div>
+          <div class="debt-total__void-cell"> </div>
```

Comment:
Expected indentation of 6 spaces but found 10 spaces  vue/html-indent<br>Require self-closing on HTML elements (<div>)         vue/html-self-closing